x-pack/legacy/plugins/reporting/server/usage/reporting_usage_collector.test.ts

```diff
@@ -51,7 +54,7 @@ function getPluginsMock(
         xpack_main: mockXpackMain,
       },
     },
-  };
+  } as any;
```

Comment:
This PR has some objects cast as `any` as a convenience to avoid importing more types from outside plugins.